recipe/meta.yaml

```diff
@@ -34,10 +40,11 @@ requirements:
     - setuptools_scm >=6.2
     - tomli
     - py-cpuinfo
+    - numpy
   run:
     - python
-    - numpy >=1.7
```

Comment:
Would keep our `numpy` constraint. Please just make sure this aligns with how it is in `numcodecs`